pkg/model/manifest.go

```diff
@@ -286,24 +299,68 @@ func GetManifestV2(manifestPath string) (*Manifest, error) {
 		}
 		manifest.Filename = path
 		return manifest, nil
-	} else if manifestPath != "" && pathExistsAndDir(manifestPath) {
+	}
+
+	return nil, nil
```

Comment:
It's a good practice to return errors in Go according to  [Effective Go](https://golang.org/doc/effective_go.html#errors), I think that you should have `return nil, CustomErr` and then check if the error is the one that you are returning. [Go blog]( https://go.dev/blog/error-handling-and-go) also has a good understanding on how to handle errors